module/behaviour/strategy/SoccerStrategy/src/SoccerStrategy.cpp

```diff
@@ -439,37 +327,174 @@ namespace module::behaviour::strategy {
     }
 
     void SoccerStrategy::goalieWalk(const Field& field, const Ball& ball) {
+        // make a motionCommand to emit
         std::unique_ptr<MotionCommand> motionCommand;
```

Comment:
I like being explicit :*(
```suggestion
        auto motionCommand = std::make_unique<MotionCommand>();
```